lib/room.cpp

```diff
@@ -443,12 +444,10 @@ class Room::Private {
         connection->saveCurrentOutboundMegolmSession(
             id, *currentOutboundMegolmSession);
 
-        const auto sessionKey = currentOutboundMegolmSession->sessionKey();
-        if(!sessionKey) {
-            qCWarning(E2EE) << "Failed to load key for new megolm session";
-            return;
-        }
-        addInboundGroupSession(currentOutboundMegolmSession->sessionId(), *sessionKey, q->localUser()->id(), "SELF"_ls);
+        currentOutboundMegolmSession->sessionKey();
```

Comment:
is the point of this line to cause a QOLM_INTERNAL_ERROR if something is broken?

Reply:
Well, I was really just converting the previous code mechanically and didn't notice ending up with a no-op. I think this should just be dropped; if something is wrong with the session key, and internal error will imminently occur whenever an attempt to use the session is made.